GreetingTest.java

```diff
@@ -0,0 +1,28 @@
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import static org.junit.Assert.assertEquals;
+
+public class GreetingTest {
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+
+    @Before
+    public void setUp() {
+        System.setOut(new PrintStream(outContent));
+    }
+
+    @After
+    public void tearDown() {
+        System.setOut(null);
```

Comment:
Basically, this "breaks" System.out after the test. The test should configure everything it needs but then restore the world to the way it was before...